docs/orchestration/concepts/services.md

```diff
@@ -35,7 +35,19 @@ Where necessary, flow runs without submitted or running task runs will be resche
 
 ## Zombie Killer
 
-The `Zombie Killer` service is responsible for handling zombies, which Prefect defines as tasks that claim to be running but haven't updated their heartbeat in the past 2 minutes.
+The `Zombie Killer` service is responsible for handling zombies, which Prefect defines as tasks that claim to be running but haven't updated their heartbeat in the past 2 minutes. It's possible for the system kernel to terminate the heartbeat process prematurely and trigger the `Zombie Killer`. To circumvent this, try configuring `Prefect` to run the heartbeat in a thread.
```

Comment:
This should note the Server heartbeat timeout as well

Reply:
I'd also put this note on circumventing in a `### Heartbeat modes` section below the description of the actual service. It feels weird to tell them that the system may misbehave while we're still explaining how the zombie killer service is useful